src/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import Routes from './Routes';
-
+import './styles/common.scss';
+import './styles/reset.scss';
```

Comment:
- common.scss 가 적용된 이후에 reset.scss 가 적용이 되기 때문에, 전체적으로 사용되는 스타일링이 다시 reset 에 의해 초기화 될 것 같습니다. reset.scss 를 먼저 import 해주세요!
```suggestion
import './styles/reset.scss';
import './styles/common.scss';
```